<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>todo</title>
    <link rel="stylesheet" href="files/bootstrap.min.css" integrity="sha384-Smlep5jCw/wG7hdkwQ/Z5nLIefveQRIY9nfy6xoR1uRYBtpZgI6339F5dgvm/e9B" crossorigin="anonymous">
    <link rel="stylesheet" href="files/choi.css" type="text/css">

    <style>
      body {
          transform: scale(0.7);
          transform-origin: top left;
          width: 142.86%; /* 100% / 0.7 to compensate for the scale */
          margin: 0;
          padding: 0;
      }
      .container {
          max-width: 142.86%; /* Adjust container width to match body width */
          padding-left: 20px; /* Add some padding on the left */
      }
      .profile-image {
          width: 200px;
          height: 200px;
          border-radius: 50%;
          object-fit: cover;
          margin-bottom: 20px;
      }
      .header-content {
          margin-bottom: 30px;
      }
      h1, h3 {
          margin-top: 20px;
          margin-bottom: 10px;
      }
  </style>
</head>
<body>
    <div class="container">
        <div class="content">
            <div class="menu-bar">
                <a href="#intro-section">Intro</a> | 
                <a href="#news-section">News</a> | 
                <a href="#research-interests-section">Research Interests</a> | 
                <a href="#publications-section">Publications</a> |
                <a href="#talks-section">Talks</a>
            </div>
<br> 
            <div class="header-content">
                <img src="files/profile-uw-2022.jpeg" alt="Cristian Øvadiuc" class="profile-image">

                <h1 >Cristian Øvadiuc  </h1>
                <p><strong>PhD Student | Genetics, University of Washington</strong></p>
                <p>[✓] MS Genetic Epidemiology, University of Washington<br>
                   [✓] BS Microbiology, University of Washington</p>

                <p>Email: {}</p>

                <address>
                    <strong>Affiliations:</strong><br>

                    <a href="https://www.fredhutch.org/en/research/divisions/vaccine-infectious-disease-division.html"> Fred Hutch -- Vaccine and Infectious Disease Division (VIDD) </a> <br>
                    University of Washington, Seattle, WA<br> </p>
                    
                  
            </div>

            <h3 id="intro-section">Introduction</h3>


            <b> 2nd year PhD </b> Public Health Genetics (<a href="https://escience.washington.edu/data-science-learning/data-science-at-uw/grad-students/grad-options/"> data science/ml option </a>) at the University of Washington, Seattle, WA. <br> <br>

<b> Completing</b> PhD  with the <a href="https://mccronelab.github.io" > McCrone Lab </a>, <a href="https://bedford.io"> Bedford Lab </a>  and  <a href="https://nextstrain.org">  Nextstrain  </a>   jointly mentored by <a href="https://www.fredhutch.org/en/faculty-lab-directory/mccrone-john.html"> Dr. JT McCrone</a> and <a href="https://www.fredhutch.org/en/faculty-lab-directory/bedford-trevor.html" >  Dr. Trevor Bedford.** </a> <br> <br>

<b> Working</b> at the nexus of computional virology + genomic epidemiology + AI/LLMs. For an overview of the field - see Trevor's terrific explanation <a href=" https://www.macfound.org/fellows/class-of-2021/trevor-bedford"> →here </a> </p> 


<b> Focusing </b> on viral dynamics across biological scales from molecular to global phylodynamics </a> linking <a href="https://www.annualreviews.org/content/journals/10.1146/annurev-virology-010320-061642"> within-host viral evolution </a> to between-host viral dynamics to inform vaccines/MCMs, viral emergence and new fundamental virology. Also interested in AI-driven predictive models such as <a href="https://arcinstitute.org/news/blog/evo"> genomic Large Language Models (LLMs) </a>  to predict viral-host compatibility and enhance <a href="https://nextstrain.org/">Nextstrain's</a> real-time tracking of pathogen evolution. </p> 


<p style="line-height: 1.5;">
    <b>Engaging</b> in high-level strategic discussions on global pandemic preparedness and biosecurity as part of the <a href="https://centerforhealthsecurity.org/education-training/emerging-leaders-in-biosecurity-fellowship">Johns Hopkins Center for Health Security - Emerging Leaders in Biosecurity (ELBI) Fellowship, Class of 2024</a>. Engaged in key sessions in Washington, DC <a href="https://www.linkedin.com/posts/jhcenterforhealthsecurity_activity-7171303534032736256-WXhZ">→here</a> with the <a href="https://www.whitehouse.gov/oppr/">White House — Office of Pandemic Preparedness and Response Policy (OPPR)</a>, led by Major General (Ret.) Paul Friedrichs. Further contributions took place in Geneva, Switzerland <a href="https://www.linkedin.com/posts/jhcenterforhealthsecurity_great-to-be-with-the-emerging-leaders-in-activity-7247681609351479298-VYNL">→here</a> at the <a href="https://www.who.int/">World Health Organization</a> and the <a href="https://www.unidir.org/">United Nations Institute for Disarmament Research</a>. A key highlight was taking on the role of National Institutes of Health (NIH) Director to provide strategic oversight in coordinating the nation's biomedical research response, guiding efforts to accelerate development of vaccines and medical countermeasures (MCMs) in a tabletop simulation exercise based on <a href="https://centerforhealthsecurity.org/our-work/tabletop-exercises/clade-x-tabletop-exercise">Clade X scenario</a>.
    </p>
    



<i> Prior, I formally trained as an epidemiologist holding a <a href="https://iphg.biostat.washington.edu/programs/ms">MS in Genetic Epidemiology</a> and worked at the <a href="https://www.cdph.ca.gov/Programs/CID/DCDC/Pages/DCDC.aspx">California Department of Public Health's Division of Communicable Disease Control</a> in addition to completing a year-long field/applied epidemiology training program – <a href="https://epi.washington.edu/students/student-epidemic-action-leaders-seal-team/">UW Student Epidemic Action Leaders (SEAL) Team</a> led by Dr. Janet Baseman through the UW Dept. of Epidemiology. Some of my research experience includes roles at the <a href="https://cerid.uw.edu/">UW Center for Emerging and Re-emerging Infectious Disease Research</a> in the <a href="https://cerid.uw.edu/people/thomas-hawn">Hawn Lab</a>, the <a href="https://www.ipd.uw.edu/">UW Institute for Protein Design</a> in the <a href="https://www.bakerlab.org/">Baker Lab***</a>, the <a href="https://www.healthdata.org/">Institute for Health Metrics and Evaluation</a>, and <a href="https://ccdd.hsph.harvard.edu/">Harvard's Center for Communicable Disease Dynamics</a>. </i> </p> 

<i>**note: prior to joining current labs -  completed PhD rotation at UW Genome Science &amp; Brotman Baty Institute for Precision Medicine w/ <a href="https://staritalab.org/"> Starita Lab </a> and <a href="https://stergachislab.org/">  Stergachis Lab </a> on <a href="https://reporter.nih.gov/search/UA5fs6tfvk2Tf0yCcdmAjA/project-details/10674373">  NIH Grant 1R01HG013025-01</a>; Conducted interviews and developed a needs assesment survey <a href="http://redcap.link/BBICVD"> →see here </a>  </i> <br> <p> <p> 

<i>***note: worked on protein structure prediction in Dr. David Baker's lab, who was awarded the 2024 Nobel Prize in Chemistry for Computational Protein Design ! <a href="https://newsroom.uw.edu/news-releases/biochemist-david-baker-receives-nobel-prize"> →see here </a>  </i> 



<br><br>
            
<h3 id="news-section">Blog:</h3>

<ul>
    
    <li><a href="blog/de-extinction-science.html">De-Extinction Science :: Dual-Use Concerns</a></li>
    </a></li>
    <!-- Add other blog posts here -->
</ul>

            
            
            <h3 id="news-section">Recent News (2021 - Present):</h3>
        
            <ul class="news-list">
                <li>Attended ELBI meeting <a href="https://www.linkedin.com/posts/jhcenterforhealthsecurity_great-to-be-with-the-emerging-leaders-in-activity-7247681609351479298-VYNL">→link</a> in Geneva, Switzerland at the <a href="https://www.who.int/"><b>World Health Organization</b></a> and <a href="https://www.unidir.org/"><b>United Nations Institute for Disarmament Research</b></a></li>

                <li>Passed PhD Preliminary Exam, June 2024</li>
                <li>Attended ELBI meeting <a href="https://www.linkedin.com/posts/jhcenterforhealthsecurity_activity-7171303534032736256-WXhZ">→link</a> in DC at the <a href="https://bidenwhitehouse.archives.gov/oppr/"> <b>White House</b> --- Office of Pandemic Preparedness and Response (OPPR) </a></li>
                <li>Joined <a href="https://centerforhealthsecurity.org/education-training/emerging-leaders-in-biosecurity-fellowship"> <b>Johns Hopkins Center for Health Security</b> - Emerging Leaders in Biosecurity (ELBI) Fellowship (Class of 2024) </a></li>
                <li><span class="highlight">Started PhD in Genetics</span> (data science/ml option) at the <b>University of Washington</b>, September 2023</li>
                <li>Graduated w/ MS Genetic Epidemiology; <a href="https://digital.lib.washington.edu/researchworks/handle/1773/50918"> <b>MS Thesis</b> --- <i>"Biological Signatures of Infectious Disease Resistance"</i> </a></li>
                <li>Lectured @ Fred Hutch<a href="https://www.fredhutch.org/en/education-training/undergraduate-students/seattlestatgrows.html"> <b>SeattleStatGROWS</b>: Undergrad Biomedical Data Science Summer Research Program</a> <a href="https://docs.google.com/presentation/d/1rC4N1Inaonkv52qPZhC36O-Wi1lvDo9-/edit?usp=sharing&amp;ouid=114259156897061838309&amp;rtpof=true&amp;sd=true">→slides here</a></li>
                <li>Joined <a href="https://cerid.uw.edu/people/thomas-hawn"> <b>Hawn Lab</b></a> at the <a href="https://cerid.uw.edu/"> Center for Emerging/Reemerging Infectious Diseases (CERID)</a> for my MS thesis</li>
                <li>Taught (TAed) for <a href="https://epi.washington.edu/course/514/">EPI 514 Application of Epidemiological Methods</a></li>
                <li>Joined <a href="https://epi.washington.edu/students/student-epidemic-action-leaders-seal-team/">UW Student Epidemic Action Leaders (SEAL) Team (Class of 2021-2022)</a>; year-long applied/field epidemiology training program focused on outbreak investigations and public health preparedness and response</li>
                <li><span class="highlight">Started MS Genetic Epidemiology</span> at the <b>University of Washington</b>, September 2021</li>
            </ul>
            

<h3>Recent Talks (2021 - Present):</h3>

<table width="100%" border="0">
<tbody style="font-size:100%">

<tr>
  <td width="80" align="left" valign="top">
June 2023
</td>
<td>
Guest Lecturer at Fred Hutch SeattleStatGROWS: Undergrad Biomedical Data Science Research Program <a href="https://docs.google.com/presentation/d/1rC4N1Inaonkv52qPZhC36O-Wi1lvDo9-/edit?usp=sharing&amp;ouid=114259156897061838309&amp;rtpof=true&amp;sd=true">→slides here</a> </td>
</tr>


<tr>
  <td width="80" align="left" valign="top">
July 2022
</td>
<td>
Talk at California Department of Public Health - Division of Communicable Disease Control  </td>
</tr>


</tbody>
</table>
<br>

</ui></ui></ui></ui></ui></ui></ui></ul>

<h3 id="publications-section">Recent Publications (2021 - Present):</h3> 
<table width="100%" border="0">
<tbody style="font-size:100%">

<tr>
  <td colspan="2"><hr></td>
</tr>

<tr>
  <td width="80" align="left" valign="top">
    2025 (in preparation)
  </td>
  <td style="padding-bottom: 1em;">
    <a href="#">
        Alveolar macrophage transcriptional signatures associated with resistance to TST/IGRA conversion following Mycobacterium tuberculosis exposure
    </a>
    <br>
    Haynes, A. M., Øvadiuc, C., Peterson, G. J., Worodria, W., Nalukwago, S., Muwanguzi, T., Nsereko, M., Stein, C. M., Dill-MacFarland, K., Mayanja-Kizza, H., Boom, W. H., Segnitz, M., & Hawn, T. R.
    <br>    <br>
    <b>Importance:</b> This study investigates transcriptional signatures in alveolar macrophages that may confer natural resistance to tuberculosis infection, providing insights into novel mechanisms of disease resistance and potential therapeutic targets.
  </td>
</tr>

<tr>
  <td colspan="2"><hr></td>
</tr>

<tr>
  <td width="80" align="left" valign="top">
    2025 (in review at AJHG)
  </td>
  <td style="padding-bottom: 1em;">
    <a href="https://www.medrxiv.org/content/10.1101/2025.01.25.25321117v1">
        Insights on improving accessibility and usability of functional data to unlock its potential for variant interpretation
    </a>
    <br>
    Park, M. S., Kumar, R. D., Øvadiuc, C., Folta, A., McEwen, A. E., Snyder, A., ... & Stergachis, A. B. (2025). <i>medRxiv</i>, 2025-01.
    <br>    <br>
    <b>Importance:</b> This survey of 190 genetics professionals revealed critical gaps in the use of functional data for clinical variant interpretation. The findings highlight the urgent need for a comprehensive database with reliable quality metrics and standardized guidelines for handling conflicting functional evidence in variant classification. <a href="http://redcap.link/BBICVD">See the survey here</a>.
  </td>
</tr>

<tr>
  <td colspan="2"><hr></td>
</tr>

<tr>
  <td width="80" align="left" valign="top">
    2023 
  </td>
  <td style="padding-bottom: 1em;">
    <a href="https://digital.lib.washington.edu/researchworks/handle/1773/50918">
        Biological Signatures of Infectious Disease Resistance: Deciphering immunological mechanisms of resistance to tuberculosis (TB) by combining host genetics and immuno-transcriptomics
    </a>
    <br>
    <i>Available from Dissertations & Theses @ University of Washington WCLP; ProQuest Dissertations & Theses Global 2023</i>
    <br>    <br>
    <b>Importance:</b> By identifying population outliers with rare human genetic mutations displaying infectious disease resistance, sequencing their genomes, and exploring the mechanisms behind their unique traits, this thesis shows an example of how we can discover new drug/vaccine targets focusing on adaptive and protective rare human genetic variation.
  </td>
</tr>

<tr>
  <td colspan="2"><hr></td>
</tr>

</tbody>
</table>

<br>

<style>
    .styled-text {
        background-color: hwb(352 25% 13%); 
        color: #fff; /* White text */
        padding: 2px 4px;
        border-radius: 3px;
    }
</style>

<h3>Long-term Research Aims:</h3>



<p class="""<b></b>  :: Reverse-engineering viral evolution / fitness landscapes mapping a path towards shifting from reactive to anticipatory defense to preempt and counter emerging high-consequence bio-threats. (see this <a href="files/paper_01.pdf">paper</a> and wikipedia entries on <a href="https://en.wikipedia.org/wiki/Fitness_landscape">fitness landscapes</a> and <a href="https://en.wikipedia.org/wiki/Viral_quasispecies">viral swarms/quasispecies</a>)</p>
</b>

:: While AI advances accelerate the reverse-engineering of viral evolution, they simultaneously create an urgent dual-use risk by lowering barriers to engineer novel threat -  the true extent of which requires immediate assessment. <br> <br> 

<p class="" style="color: rgb(1, 1, 1);"<p> :: This requires a phased approach w/ <span class="styled-text">(A) RED TEAMING AI Models for vulnerability mapping,</span> (B) development of AI-specific biothreat models and technical safeguards, and (C) informed policy development based on mitigation assessments and expert engagement. National Academies of Science workshop/publication on Assessing and Navigating Biosecurity Concerns and Benefits of Artificial Intelligence Use in the Life Sciences provides additional context.</b> ( <a href="https://www.nationalacademies.org/our-work/assessing-and-navigating-biosecurity-concerns-and-benefits-of-artificial-intelligence-use-in-the-life-sciences" style="color: rgb(0, 0, 0);"> --> read here </a> ) 
    <br></p>
::Developing a Biothreat/Biodefense AI Co-Pilot analogous to AI coding assistants -  might offer a significant opportunity to enhance capacity for processing viral genomes and complex outbreak data. 

 
 </p>

<img src="files/demo3.gif" alt="Research Aims Demonstration" style="max-width: 100%; height: auto;">

    
<br><br>
<br><br>
            <footer>
            Website theme source: <a href="https://homes.cs.washington.edu/~yejin/">Yejin Choi</a></p>
            </footer>
            <br><br>
            <br><br>
            <br><br>
            <br><br>
            <br><br>
            <br><br>

        </div>
    </div>
</body>
</html>
